rfcs/45-stageless.md

```diff
@@ -184,36 +227,34 @@ fn main() {
         .add_plugins(DefaultPlugins)
         // You can add functions with read-only system parameters as conditions.
         .add_set(GameSet::Construction.run_if(construction_timer_not_finished))
-        .add_many(systems![tick_construction_timer, update_construction_progress].in_set(GameSet::Construction))
-        .add_system(system_meltdown_mitigation.run_if(too_many_enemies))
-        // We can use closures for simple one-off conditions, 
-        // which automatically fetch the appropriate data from the `World`.
+        .add_systems(
+            systems![tick_construction_timer, update_construction_progress]
+                .in_set(GameSet::Construction))
+        .add_system(mitigate_meltdown.run_if(too_many_enemies))
+        // We can use closures for simple one-off conditions.
         .add_system(spawn_more_enemies.run_if(|difficulty: Res<Difficulty>| difficulty >= 9000))
-        // The `resource_exists` and `resource_equals` helper functions for generating
-        // simple conditions (well, closures that convert into conditions).
+        // `resource_exists` and `resource_equals` are helper functions that capture
+        // variables in simple closures that can be converted into conditions.

```

Comment:
```suggestion
        // `resource_exists` and `resource_equals` are helper functions which produce
        // new systems that can be used as run conditions.

```

Reply:
I changed "systems" to "closures", (just trying to avoid any confusion), but this is good.